test/Manifest.toml

```diff
@@ -0,0 +1,319 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
For packages that you expect others to use, you generally need not commit the Manifest.toml. I would prefer that we not commit it because it will just produce unnecessary hassle.

Reply:
Looks like I forgot to add it to .gitignore. Fixed